server/src/main/java/org/opensearch/action/support/clustermanager/TransportClusterManagerNodeAction.java

```diff
@@ -218,7 +222,7 @@ protected void doStart(ClusterState clusterState) {
                             }
                         });
                         threadPool.executor(executor)
-                            .execute(ActionRunnable.wrap(delegate, l -> clusterManagerOperation(task, request, clusterState, l)));
+                            .execute(ActionRunnable.wrap(delegate, l -> masterOperation(task, request, clusterState, l)));
```

Comment:
May be we could keep the `clusterManagerOperation` but change its delegation model:

```
    protected void clusterManagerOperation(Task task, Request request, ClusterState state, ActionListener<Response> listener)
        throws Exception {
        masterOperation(task, request, state, listener);
    }
```

Reply:
Changed in the commit https://github.com/opensearch-project/OpenSearch/pull/4103/commits/d4ec950140daf527f8b4988a1aaa61485263b860, thanks for your brilliant idea! 😄